boards/arm/nrf52_pca20044/Kconfig.defconfig

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+
+if BOARD_NRF52_PCA20044
+
+config BOARD
+	default "nrf52_pca20044"
+
```

Comment:
Don't you need to add
```
config BT_CTLR
	default BT
```
here?

Reply:
beats me...  Honestly I don't know. 20041 which works perfectly fine does not have it.

BT_CTLR is defined by BLE controller kconfig so I don't know why boards add it again or why parser does not complain it is redefined.

Since you asked about it I add it but I don't know what it is used for.

Reply:
This is to enable BLE controller by default when Bluetooth support is enabled. Sometimes you may want to not include the controller code in the build but instead use some other chip via HCI over some interface.
Perhaps BT_CTLR is enabled in the application's configuration and that's why it works?

Reply:
yes, we add it as we use the controller. 
anyway - it is now added as for other boards.